test/onnx/test_models_onnxruntime.py

```diff
@@ -245,6 +245,7 @@ def test_faster_rcnn(self):
             atol=1e-5,
         )
 
+    @unittest.skip("Failing after ONNX 1.13.0")
```

Comment:
Is this a regression?

Reply:
Still investigating this. It fails on the assertion error introduced in #87620